nautobot_floor_plan/utils.py

```diff
@@ -25,6 +25,56 @@ def grid_letter_to_number(letter):
     return number
 
 
+def axis_init_label_conversion(axis_origin, axis_location, step, letters):
```

Comment:
Let's update the `letters` variables in these functions to be `is_letters` as well.